tests/test_read_simple_bitext.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+import pytest
+
+from nemo_curator.datasets.doc_dataset import ParallelDataset
+
+
+@pytest.fixture
+def src_file():
+    return Path("tests/bitext_data/toy.de")
+
+
+@pytest.fixture
+def tgt_file():
+    return Path("tests/bitext_data/toy.en")
+
+
+class TestReadSimpleBitext:
+
+    def test_pandas_read_simple_bitext(self, src_file, tgt_file):
+        ds = ParallelDataset.read_simple_bitext(
+            src_input_files=[src_file],
+            tgt_input_files=[tgt_file],
+            src_lang="de",
+            tgt_lang="en",
+            backend="pandas",
+        )
+
+        for idx, (src_line, tgt_line) in enumerate(zip(open(src_file), open(tgt_file))):
+            assert ds.df["src"].compute()[idx] == src_line.rstrip("\n")
```

Comment:
Nit: instead of calling compute multiple times, call it once and then do all the assert statements.

Reply:
Interestingly, I get errors when I call it only once:

```
TypeError: Trying to convert dd.Scalar<series-..., dtype=bool> to a boolean value. Because Dask objects are lazily evaluated, they cannot be converted to a boolean value or used in boolean conditions like if statements. Try calling .compute() to force computation prior to converting to a boolean value or using in a conditional statement.
```